src/components/sharing/util.js

```diff
@@ -407,3 +409,41 @@ export const getUnsharingUpdates = (originalUsers, userMap) => {
         return acc;
     }, {});
 };
+
+export const formatSharedData = (selectedData) => {
+    const formattedData =
+        selectedData?.map((data) => {
+            return {
+                label: data.label,
+                path: data.path,
+                type: data.type,
+            };
+        }) || [];
```

Comment:
Destructuring could make this a little more condensed:
```suggestion
        selectedData?.map(({ label, path, type }) => ({ label, path, type })) || [];
```